packages/monaco/src/typings/file-icons-js.d.ts

```diff
@@ -0,0 +1,20 @@
+/********************************************************************************
```

Comment:
i think we have these typings already in one place, why to duplicate here?

Reply:
I will try to remove it, but for some reason the build kept failing saying it was missing such a file.